PyPDF2/_merger.py

```diff
@@ -75,20 +76,34 @@ class PdfMerger:
     See the functions :meth:`merge()<merge>` (or :meth:`append()<append>`)
     and :meth:`write()<write>` for usage information.
 
+    :param fileobj: Output file. Can be a filename or any kind of
+            file-like object.
     :param bool strict: Determines whether user should be warned of all
             problems and also causes some correctable problems to be fatal.
             Defaults to ``False``.
     """
 
-    def __init__(self, strict: bool = False) -> None:
+    def __init__(self, fileobj: StrByteType = "", strict: bool = False) -> None:
```

Comment:
Per the earlier BC discussion, `fileobj` should come after `strict` or else it breaks usage for people that were doing `PdfWriter(True)` or whatever. In the docs, we should use a named attribute when referring to this so that in say v3, we can swap the ordering to what's here without worrying about things breaking for people.

```suggestion
    def __init__(self, strict: bool = False, fileobj: StrByteType = "") -> None:
```